core/components/header/index.tsx

```diff
@@ -143,16 +144,17 @@ export const Header = async () => {
         searchHref: '/search',
         searchParamName: 'term',
         searchAction: search,
-        links: Streamable.from(getLinks),
-        logo: Streamable.from(getLogo),
+        links: streamableLinks,
+        logo,
         mobileMenuTriggerLabel: t('toggleNavigation'),
         openSearchPopupLabel: t('Icons.search'),
         logoLabel: t('home'),
-        cartCount: Streamable.from(getCartCount),
+        cartCount: streamableCartCount,
         activeLocaleId: locale,
         locales,
         currencies,
-        activeCurrencyId,
+        // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
Why not update the component so that it accepts a streamable for `activeCurrencyId`?

Reply:
Good catch! Missed this.